src/migrations/20171005123235_markets.ts

```diff
@@ -25,6 +25,8 @@ exports.up = async (knex: Knex): Promise<any> => {
       "feeWindow" varchar(66),
       "endTime" integer NOT NULL CONSTRAINT "positiveEndTime" CHECK ("endTime" > 0),
       "finalizationTime" integer,
+      "forking" boolean DEFAULT O,
```

Comment:
I think this may need to be changed. It looks like this is defaulting to the letter "O" instead of zero?

Reply:
🤦‍♂️